homeassistant/components/smhi/sensor.py

```diff
@@ -89,6 +104,89 @@ class SMHISensorEntityDescription(SensorEntityDescription):
         value_fn=lambda entity: get_percentage_values(entity, "frozen_precipitation"),
         native_unit_of_measurement=PERCENTAGE,
     ),
+    # Fire sensors
+    SMHISensorEntityDescription(
+        key="fwiindex",
+        translation_key="fwiindex",
+        value_fn=lambda entity: get_fire_index_value(entity, "fwiindex"),
+        device_class=SensorDeviceClass.ENUM,
+        options=["1", "2", "3", "4", "5", "6"],
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
+    ),
+    SMHISensorEntityDescription(
+        key="fwi",
+        translation_key="fwi",
+        value_fn=lambda entity: entity.coordinator.fire_current.get("fwi"),
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_registry_enabled_default=False,
+    ),
+    SMHISensorEntityDescription(
+        key="isi",
+        translation_key="isi",
+        value_fn=lambda entity: entity.coordinator.fire_current.get("isi"),
```

Comment:
would it make sense to have more descriptive translation keys? I think that improves reading the code and probably also helps translators a bit